geowebcache/web/src/main/webapp/WEB-INF/geowebcache-sqlite-context.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:mvc="http://www.springframework.org/schema/mvc"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+                           http://www.springframework.org/schema/beans/spring-beans.xsd
+                           http://www.springframework.org/schema/context
+                           http://www.springframework.org/schema/context/spring-context-3.0.xsd
+                           http://www.springframework.org/schema/mvc
+                           http://www.springframework.org/schema/mvc/spring-mvc-3.1.xsd">
+    <mvc:annotation-driven/>
```

Comment:
This is the only spring context file enabling use of annotations, as far as I can say, they are being used for the rest operations only. Personally I don't have a case against using this approach, but I'd at least check with the rest of devs on the devel list if going this way is welcomed, as it might introduce some learning curve for the core developers.